Extensions/Lighting/forwarddiffusedRenderFilter.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * A pixi filter for diffused rendering. 
+ * @param {gdjs.RuntimeSceneRenderer} runtimeSceneRenderer 
+ */
+gdjs.ForwardDiffusedRenderFilter = function (runtimeSceneRenderer) {
```

Comment:
@4ian how do I import this class? It doesn't fit as an object, behavior or effect, how do I use JsExtension in this case?

Reply:
As it's part of an extension, it should be included by the extension whenever needed, which is surely whenever one of the new "light" object is used. As you don't have any new "light" object for now, consider either:
- "hacking" you way by adding `addIncludeFile` to some action, condition or existing object. Don't forget to put a big TODO
- OR maybe this filter should be part of the game engine itself when using pixi? In this case, add it in `GDJS/Runtime/pixi-renderers` and list it in `ExporterHelper::AddLibsInclude` in ExporterHelper.cpp. Though this means every game will include it even if not used, so maybe less ideal... but maybe we'll still need this because the rendering of normal map lighting will be tightly integrated in the pixi game engine.